### PR TITLE
Correct paid content cAPI template fixed

### PR DESCRIPTION
**Pull request by Calanthe**
cc @kelvin-chappell

### CODE REVIEW

File: commercial/app/views/paidContent/card.scala.html [hunk 1]
```diff
@@ -17,9 +17,13 @@
         </a>
     </div>
     @fragments.commercial.paidForMeta()
-    <a href="@optClickMacro@articleUrl" class="paidfor-box" data-link-name="advertisers logo">
+    <a href="<%=clickMacro%><%=articleUrl%>" class="paidfor-box paidfor-box--header" data-link-name="advertisers header">
```
Comment (kelvin-chappell) [inline commercial/app/views/paidContent/card.scala.html]: In a scala template href will still need to be `@optClickMacro@articleUrl` I think.

File: commercial/app/views/paidContent/card.scala.html [hunk 2]
```diff
@@ -17,9 +17,13 @@
...
         <h2 class="gu-title">@articleTitle</h2>
-            @for(text <- articleText){ <div class="gu-text">@Html(text)</div> }
+    </a>
+    <a href="<%=clickMacro%><%=articleUrl%>" class="paidfor-box paidfor-box--text" data-link-name="advertisers text">
+        @for(text <- articleText){ <div class="gu-text">@Html(text)</div> }
+    </a>
+    <a href="<%=clickMacro%><%=articleUrl%>" class="paidfor-box paidfor-box--logo" data-link-name="advertisers logo">
```
Comment (kelvin-chappell) [inline commercial/app/views/paidContent/card.scala.html]: And here.

File: commercial/app/views/paidContent/card.scala.html [hunk 3]
```diff
@@ -17,9 +17,13 @@
         </a>
     </div>
     @fragments.commercial.paidForMeta()
-    <a href="@optClickMacro@articleUrl" class="paidfor-box" data-link-name="advertisers logo">
+    <a href="<%=clickMacro%><%=articleUrl%>" class="paidfor-box paidfor-box--header" data-link-name="advertisers header">
         <h2 class="gu-title">@articleTitle</h2>
-            @for(text <- articleText){ <div class="gu-text">@Html(text)</div> }
+    </a>
+    <a href="<%=clickMacro%><%=articleUrl%>" class="paidfor-box paidfor-box--text" data-link-name="advertisers text">
```
Comment (kelvin-chappell) [inline commercial/app/views/paidContent/card.scala.html]: Same here.

File: commercial/app/views/paidContent/card.scala.html (hunk 1 above)
Comment (Calanthe) [inline commercial/app/views/paidContent/card.scala.html]: yeah, ofc you are right

**Comment by kelvin-chappell** [comment]
:+1:

**Pull request merged**